### PR TITLE
English translated docs version 09.06.20

### PR DESCRIPTION
**Pull request by xxxpsyduck**

### CODE REVIEW

**Comment by LDOUBLEV** [comment]
Thank you very much for your contribution to PaddleOCR repo. Your completed OCR English document will expand the spread of PaddleOCR. We always look forward to developers like you joining us. 

However,  to merge this PR, you also need to sign the CLA agreement to pass the PR check,  don't worry, this will not have any adverse effects on you but accept your contribution.

image

File: README.md [hunk 1]
```diff
@@ -1,163 +1,163 @@
-## 简介
-PaddleOCR旨在打造一套丰富、领先、且实用的OCR工具库，助力使用者训练出更好的模型，并应用落地。
-
-**近期更新**
-- 2020.6.8 添加[数据集](./doc/datasets.md)，并保持持续更新
-- 2020.6.5 支持 `attetnion` 模型导出 `inference_model`
-- 2020.6.5 支持单独预测识别时，输出结果得分
-- 2020.5.30 提供超轻量级中文OCR在线体验
-- 2020.5.30 模型预测、训练支持Windows系统
+## Introduction
```
Comment (LDOUBLEV) [inline README.md:1]: It is recommended to put the English readme in the README_EN.md file, to distinguish it from the Chinese README

**Comment by xxxpsyduck** [comment]
> Thank you very much for your contribution to PaddleOCR repo. Your completed OCR English document will expand the spread of PaddleOCR. We always look forward to developers like you joining us.
> 
> However, to merge this PR, you also need to sign the CLA agreement to pass the PR check, don't worry, this will not have any adverse effects on you but accept your contribution.
> 
> image

Hi, this github account already signed to the CLA. I don't know why it's showing this warning. Maybe due to I use different email in git config in my local machine? Any recommendation to get rid of it?

**Comment by xxxpsyduck** [comment]
@LDOUBLEV Should I delete current fork and create a new one?

**Comment by tink2123** [comment]
You don't need to delete it. If you have changed your user.name, you can simply use push xxx -f to resolve this problem

**Comment by xxxpsyduck** [comment]
@tink2123 thanks I got it resolved. Can you delete this pull request?

**Comment by tink2123** [comment]
@xxxpsyduck Sorry, Deleting PR is not supported on Github. Close is ok，don't worry

**Pull request closed without merging**